app/controllers/org_admin/templates_controller.rb

```diff
@@ -269,13 +262,13 @@ def publish
     def unpublish
       template = Template.find(params[:id])
       authorize template
-      if template.latest?
-        template.published = false
-        template.save!
-        flash[:notice] = _("Your #{template_type(template)} is no longer published. Users will not be able to create new DMPs for this #{template_type(template)} until you re-publish it")
-      else
-        flash[:alert] = _("That #{template_type(template)} is not currently published.")
+      versions = Template.where(family_id: template.family_id)
+      versions.each do |version|
```

Comment:
why do we want this https://github.com/DMPRoadmap/roadmap/blob/sprint2/app/models/template.rb#L8 then?

Reply:
Because the user is always presented with the latest version in the templates table. The desire is to allow an admin to unpublish a template whether or not the published version is the latest.

e.g. I am an admin and start editing my published template which creates a draft version. I then realize that I want to unpublish the 'live' one. We currently offer no way to do that.